src/Build/BackEnd/Components/Logging/ILoggingService.cs

```diff
@@ -555,6 +555,20 @@ BuildEventContext LogProjectStarted(
             int evaluationId = BuildEventContext.InvalidEvaluationId,
             int projectContextId = BuildEventContext.InvalidProjectContextId);
 
+        void LogProjectStarted(ProjectStartedEventArgs args);
```

Comment:
This is actually logged immediately - we just do not log it 'implicitly' in the depths of `NodeLoggingContext.LogProjectStarted`, but instead create the event explicitly - so that additional data can be attached - and then immediately log